cryptographic_estimators/SDFqEstimator/SDFqAlgorithms/stern.py

```diff
@@ -156,7 +156,7 @@ def _time_and_memory_complexity(self, parameters: dict, verbose_information=None
         if verbose_information is not None:
             verbose_information[VerboseInformation.PERMUTATIONS.value] = Tp
             verbose_information[VerboseInformation.GAUSS.value] = log2(Tg)
-            verbose_information[VerboseInformation.LISTS.value] = [log2(L2), log2(L)]
+            verbose_information[VerboseInformation.LISTS.value] = [log2(L2), log2(L), log2(ops), log2(ops + Tg), solutions]
```

Comment:
these are no lists

Reply:
jeah, need to be removed. I added them for debugging